### PR TITLE
Adjust tabs styles

### PR DESCRIPTION
**Pull request by Aideann**
I've adjusted tabs styled according to designs and internal discussion on slack: https://livechat.slack.com/archives/CA945NPQR/p1561102438000100

Designs: https://app.zeplin.io/project/5c4ec1443fcf8345e74aafb5/screen/5c6edc6a6d6dc44097589998

### CODE REVIEW

File: src/components/Tab/style.scss [hunk 1]
```diff
@@ -40,11 +39,12 @@
     font-weight: 700;
 
     .tab__description {
-      color: $font-primary-color;
```
Comment (dprzybylek) [inline src/components/Tab/style.scss]: 😱 😱 😱

Comment (Aideann) [reply src/components/Tab/style.scss]: yes! that's how we do it! :D

File: src/components/Tab/style.scss [hunk 2]
```diff
@@ -6,19 +6,18 @@
...
   transition: color 0.25s;
-  margin: 0 8px;
-  font-size: 15px;
+  margin: 0 8px 11px 8px;
+  font-size: 14px;
 
   &__description {
     display: inline-block;
-    color: #000;
+    color: #8e949a;
```
Comment (Aideann) [inline src/components/Tab/style.scss]: After discussion with @kamilmateusiak he told me to put it directly as there are some changes planned for colors variables anyway

File: src/components/Tab/style.scss [hunk 3]
```diff
@@ -40,11 +39,12 @@
     font-weight: 700;
 
     .tab__description {
-      color: $font-primary-color;
+      color: #8e949a;
```
Comment (kamilmateusiak) [inline src/components/Tab/style.scss]: any link to design?

Comment (Aideann) [reply src/components/Tab/style.scss]: Sure: https://app.zeplin.io/project/5c4ec1443fcf8345e74aafb5/screen/5c6edc6a6d6dc44097589998

File: src/components/Tab/style.scss [hunk 4]
```diff
@@ -6,19 +6,18 @@
   flex-direction: row;
   align-items: center;
   padding: 0 24px;
-  height: 49px;
```
Comment (kamilmateusiak) [inline src/components/Tab/style.scss]: I wonder why it was static. I hope it was not relative with any IE/EDGE issue ;x

File: src/components/Tab/style.scss [hunk 5]
```diff
@@ -40,11 +39,12 @@
...
 
     .tab__description {
-      color: $font-primary-color;
+      color: #8e949a;
     }
 
     &::before {
-      height: 5px;
+      height: 4px;
+      bottom: -12px;
```
Comment (kamilmateusiak) [inline src/components/Tab/style.scss]: this line won't be needed if you add padding-bottom to the tab instead of margin

Comment (kamilmateusiak) [reply src/components/Tab/style.scss]: good point! :D

File: src/components/Tab/style.scss [hunk 6]
```diff
@@ -6,19 +6,19 @@
...
   background: none;
   outline: none;
   color: $font-primary-color;
   cursor: pointer;
   transition: color 0.25s;
-  margin: 0 8px;
-  font-size: 15px;
+  margin: 0 8px 11px 8px;
+  font-size: 14px;
+  overflow: visibile;
```
Comment (kamilmateusiak) [inline src/components/Tab/style.scss]:
```suggestion
overflow: visible;
```

File: src/components/Tab/style.scss (hunk 4 above)
Comment (Aideann) [reply src/components/Tab/style.scss]: Checked and looks fine after discussion with @kamilmateusiak

**Pull request merged**